api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -67,19 +73,29 @@ public void saveAlert_shouldSaveAlertToDb() {
 	@Test
 	public void getAlert_shouldGetAlertById() {
 		Alert savedAlert = hibernateAlertDAO.getAlert(2);
+		savedAlert.setDateToExpire(Date.from(ZonedDateTime.now().plusYears(1).toInstant()));
 		Assertions.assertEquals((int) savedAlert.getAlertId(), 2);
 	}
 
 	@Test
 	public void deleteAlert_shouldReturnNullAfterDeleting() {
 		Alert savedAlert = hibernateAlertDAO.getAlert(2);
+		savedAlert.setDateToExpire(Date.from(ZonedDateTime.now().plusYears(1).toInstant()));
 		Assertions.assertNotNull(savedAlert);
 		hibernateAlertDAO.deleteAlert(savedAlert);
 		Assertions.assertNull(hibernateAlertDAO.getAlert(2));
 	}
 
 	@Test
 	public void getAllAlerts_shouldReturnOnlyNonExpiredAllerts() {
+		Alert alert = hibernateAlertDAO.getAlert(2);
+		Date currentDate = new Date();
+		dateFormat.format(currentDate);
+		Calendar c = Calendar.getInstance();
+		c.setTime(currentDate);
+		c.add(Calendar.YEAR,1);
+		Date currentDatePlusOneYear = c.getTime();
+		alert.setDateToExpire(currentDatePlusOneYear);
```

Comment:
Couldn't we do the same thing here we did above, e.g., `alert.setDateToExpire(Date.from(ZonedDateTime.now().plusYears(1).toInstant()));`?